shared_libs/transformer_base_lib/transformer_base_lib/run_transformer.py

```diff
@@ -17,7 +17,7 @@
 if TYPE_CHECKING:
     from .base_transformer import BaseTransformer
 
-logging.basicConfig(level=os.environ.get("LOG_LEVEL", "ERROR").upper())
+logging.basicConfig(level=os.environ.get("LOG_LEVEL", "INFO").upper())
```

Comment:
the default is ERROR across our other apps, and whilst useful in dev, do we want the more detailed logs on production just for the transformers as a fallback? @ikrambapu thoughts?

the INFO ones I can see in this repo are:
```python
logger.info(
            "%s Transformer initialised with max_batch_size=%s (the parsed max_batch_size config value=%s)",
            transformer.transformer_name,
            batch_size,
            config.MAX_BATCH_SIZE,
        )

logger.info(
            "%s Transformer processor started.",
            transformer.transformer_name,
        )
```
Seem useful and non-polluting only triggering on instantiation of the container, so I'd be in favour of this change.